summary.py

```diff
@@ -219,8 +221,8 @@ def check_and_sort_regressions(regressions):
     # check we can find a lef file, which is needed for viewing def files
     lef_path = os.path.join(run_path, 'tmp', 'merged_unpadded.lef')
     if not os.path.exists(lef_path):
-        print("no LEF file found, any views that use DEF files (floorplan, pdn, fine and detailed placement) will fail")
-        
+        print("No LEF file found, any views that use DEF files (floorplan, pdn, fine and detailed placement) will fail")
+        print("Make sure you check openlane_summary out to the right MPW branch (git checkout mpw7).")
```

Comment:
please change this to something like: "make sure the summary tool is on the correct branch for your tools installation. ". As if we reference mpw7 then I'll have to update it every new tool tag.